atom-jobs/build-common.groovy

```diff
@@ -622,6 +627,17 @@ else
         mkdir -p ${TARGET}
         mv release-centos7/tiflash ${TARGET}/tiflash
     fi
+    
+    if ["${params.TIFLASH_DEBUG}" == "true" ]; then
```

Comment:
if `TIFLASH_DEBUG ` is `true`, will build tiflash twice: release mode and debug mode. build tiflash in release mode is unnecessary